pkg/cli/mirror/create.go

```diff
@@ -117,17 +125,33 @@ func (o MirrorOptions) Create(ctx context.Context, flags *pflag.FlagSet) error {
 		return err
 	}
 
-	// Run archiver
+	// Catch any cancel signals
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	done := make(chan bool, 1)
+	go func() {
+		<-sigs
+		if err := rollbackMeta(); err != nil {
+			logrus.Error(err)
+		}
+		os.Exit(1)
+		done <- true
```

Comment:
If the parent goroutine does not wait on `done` then this goroutine may not complete if triggered.